### PR TITLE
fix: bad pack cmd exec

### PR DESCRIPTION
**Pull request by matejvasek**
Signed-off-by: Matej Vasek <mvasek@redhat.com>

### CODE REVIEW

**Review by lance — APPROVED** [review]
/lgtm

**Comment by lance** [comment]
@matejvasek you should export the `PACK_CMD` env var [here](https://github.com/boson-project/buildpacks/blob/bf574788acb63acb08a8238089a513ca046d8401/Makefile#L69) I think.

**Comment by matejvasek** [comment]
> @matejvasek you should export the `PACK_CMD` env var [here](https://github.com/boson-project/buildpacks/blob/bf574788acb63acb08a8238089a513ca046d8401/Makefile#L69) I think.

@lance  I added proper export here:
https://github.com/boson-project/buildpacks/pull/107/files#diff-944291df2c9c06359d37cc8833d182d705c9e8c3108e7cfe132d61a06e9133ddR52

**Comment by matejvasek** [comment]
and I miss one more export in ci.yaml

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -49,13 +49,13 @@ jobs:
           tar -xvf pack.tgz
 
       - name: Build
-        run: PACK_CMD=./pack make
+        run: PACK_CMD="$(pwd)/pack" make
```
Comment (matejvasek) [inline .github/workflows/ci.yaml:52]: Or maybe I could just install `pack` to `/usr/local/bin` so it would be on PATH.

**Comment by matejvasek** [comment]
@lance `pack` seems to be executed now PTAL

**Pull request merged**